File: backend/src/main/java/com/example/pension/model/Dictionary.java
package com.example.pension.model;

import lombok.Data;
import java.time.LocalDateTime;

@Data
public class Dictionary {
    private Long id;
    private String dictType; // 字典类型：community(所属社区), pensionType(养老类型)
    private String dictCode; // 字典编码
    private String dictLabel; // 字典标签
    private String dictValue; // 字典值
    private Integer sortOrder; // 排序
    private String status; // 状态：ACTIVE(启用), INACTIVE(禁用)
    private String remark; // 备注
    private LocalDateTime createTime;
    private LocalDateTime updateTime;
} 

File: frontend/src/constants/dictionary.js
// 字典类型常量
export const DICT_TYPES = {
  COMMUNITY: 'community',
  PENSION_TYPE: 'pensionType'
}

// 字典类型名称映射
export const DICT_TYPE_NAMES = {
  [DICT_TYPES.COMMUNITY]: '所属社区',
  [DICT_TYPES.PENSION_TYPE]: '养老类型'
}

// 字典状态常量
export const DICT_STATUS = {
  ACTIVE: 'ACTIVE',
  INACTIVE: 'INACTIVE'
}

// 字典状态名称映射
export const DICT_STATUS_NAMES = {
  [DICT_STATUS.ACTIVE]: '启用',
  [DICT_STATUS.INACTIVE]: '禁用'
}

// 字典状态选项
export const DICT_STATUS_OPTIONS = [
  { label: '启用', value: DICT_STATUS.ACTIVE },
  { label: '禁用', value: DICT_STATUS.INACTIVE }
] 